packages/loot-core/src/server/accounts/export-to-csv.ts

```diff
@@ -75,23 +77,46 @@ export async function exportQueryToCSV(query) {
       .options({ splits: 'all' }),
   );
 
-  const parentsPayees = new Map();
+  // initialize a map to allow splits to have correct number of split from
+  const parentsChildCount = new Map();
   for (const trans of transactions) {
     if (trans.IsParent) {
-      parentsPayees.set(trans.Id, trans.Payee);
+      parentsChildCount.set(trans.Id, 0);
     }
   }
 
-  // filter out any parent transactions
-  const noParents = transactions.filter(t => !t.IsParent);
+  // sort order for children is negative number in their sort order, so invert when mapping
+  // count how many children each parent has by incrementing for each child found
+  const childSplitOrder = new Map();
+  for (const trans of transactions) {
+    if (trans.IsChild) {
+      childSplitOrder.set(trans.Id, trans.SortOrder * -1);
+      parentsChildCount.set(
+        trans.ParentId,
+        parentsChildCount.get(trans.ParentId) + 1,
+      );
+    }
+  }
 
-  // map final properties for export and grab the payee for splits from their parent transaction
-  const transactionsForExport = noParents.map(trans => {
+  // map final properties for export and grab the child count for splits from their parent transaction
+  const transactionsForExport = transactions.map(trans => {
     return {
       Account: trans.Account,
       Date: trans.Date,
-      Payee: trans.ParentId ? parentsPayees.get(trans.ParentId) : trans.Payee,
-      Notes: trans.Notes,
+      Payee: trans.Payee,
+      Notes: trans.IsParent
+        ? '(SPLIT INTO ' +
+          parentsChildCount.get(trans.Id) +
+          ') ' +
+          (trans.Notes || '')
+        : trans.IsChild
+          ? '(SPLIT ' +
+            childSplitOrder.get(trans.Id) +
```

Comment:
Add +1 to make it human readable rather than 0 based. Currently we're getting `(SPLIT 0 of 3)`

Reply:
are you using the test/demo data or real data?

I was getting (SPLIT 0... when using test/demo data, but when I generated new transactions or used my real data, it was working as intended (split 1 being first).

My database shows splits as having transaction numbers "-1, -2, -3, etc." (for my real data). Is this not the standard?

I started from looking at my real data and working backwards so maybe I made a wrong conclusion.

Reply:
Ok, sounds good. I can implement that instead.

Should the mock data get updated or not a big deal because it's just a workaround, and if/when this changes, that change could be implemented?

Reply:
implemented this. 

i *think* we need two loops here to first make sure we have all the parents are have their mapping existing before going through and giving children numbers? but i tried to combine what was three into two.